### DISCUSSION TITLE
How to package the source code into an EXE file that can run independently

### DISCUSSION THREAD

**Discussion by xianxian0703** [rustdesk/rustdesk#523] [General]
How to package the source code into an EXE file that can run independently

**Comment by xianxian0703** [comment]
wait online

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/master/build.py
but Sciter static lib is not free, I bought from sciter.com. So my build instructions can only make dynamic link version.
We miss you a GitHub star, :).

**Comment by youhers** [comment]
The same question

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/695
